src/data_rate.proto

```diff
@@ -3,29 +3,29 @@ syntax = "proto3";
 package helium;
 
 enum data_rate {
-    SF12BW125 = 1;
-    SF11BW125 = 2;
-    SF10BW125 = 3;
-    SF9BW125 = 4;
-    SF8BW125 = 5;
-    SF7BW125 = 6;
-    SF12BW250 = 7;
-    SF11BW250 = 8;
-    SF10BW250 = 9;
-    SF9BW250 = 10;
-    SF8BW250 = 11;
-    SF7BW250 = 12;
-    SF12BW500 = 13;
-    SF11BW500 = 14;
-    SF10BW500 = 15;
-    SF9BW500 = 16;
-    SF8BW500 = 17;
-    SF7BW500 = 18;
-    LRFHSS1BW137 = 19;
-    LRFHSS2BW137 = 20;
-    LRFHSS1BW336 = 21;
-    LRFHSS2BW336 = 22;
-    LRFHSS1BW1523 = 23;
-    LRFHSS2BW1523 = 24;
-    FSK50 = 25;
+    SF12BW125 = 0;
```

Comment:
One more time. Field numbers start at 1

Reply:
Reverted per: https://developers.google.com/protocol-buffers/docs/proto3#default

"every enum definition must contain a constant that maps to zero as its first element."

Reply:
No. that's the_encoded_ value in language mapping. I shared this last night already <https://developers.google.com/protocol-buffers/docs/proto3#assigning_field_numbers>

Reply:
Is there a difference in numbering for messages vs enums? My IDE was displaying an error when I removed the 0 count, which made me take a second look.

Reply:
double/triple checked. Enums values have to have a `0`.

Reply:
what.. do we do this everywhere? I must've blanked this from my memory if so

Reply:
I clearly did.. apologies <https://developers.google.com/protocol-buffers/docs/proto3#enum> specs the enum requirements